src/components/Questionnaire/QuestionnaireEditor.tsx

```diff
@@ -736,9 +770,9 @@ export default function QuestionnaireEditor({ id }: QuestionnaireEditorProps) {
     return (
       <Alert variant="destructive">
         <CareIcon icon="l-exclamation-circle" className="h-4 w-4" />
-        <AlertTitle>Error</AlertTitle>
+        <AlertTitle>{t("error")}</AlertTitle>
         <AlertDescription>
-          Failed to load questionnaire. Please try again later.
+          {t("failed_to_load_questionnaire_Please_try_later")}
```

Comment:
minor thing (update en.json as well).

```suggestion
          {t("failed_to_load_questionnaire_please_try_later")}
```